### ISSUE TITLE
After installing plugin, PHPStorm loads with error

### ISSUE THREAD

**Issue by BardiaAfshin** [fioan89/sourcesync#18]
This is a very similar behavior explained here https://youtrack.jetbrains.com/issue/WI-21507

I tried deleting the directory ./Web... under my home directory, as suggested in the thread and nothing changed. Rebooting phpstorm gave me the same error after, and the plugin doesn't show up in the tools menu.

Here's some breadcrumbs from the log file
```log
11:03:33 AM PluginException: cannot create class "org.wavescale.sourcesync.action.ActionSourceSyncMenu" [Plugin: org.wavescale.sourcesync]
11:03:42 AM PluginException: cannot create class "org.wavescale.sourcesync.action.ActionProjectConnectionConfig" [Plugin: org.wavescale.sourcesync]
```

PHPStorm version 8.0.3 build ps-139.1348

**Comment by asebak** [comment]
Yes because the library is using java 8 so if java 8 is not the default java on your o/s it loads with error

**Comment by BardiaAfshin** [comment]
I just installed Java 8, restarted PHPStorm, and the same error.

```log
12:07:47 PM PluginException: cannot create class "org.wavescale.sourcesync.action.ActionLocalFileToRemote" [Plugin: org.wavescale.sourcesync]
12:07:56 PM PluginException: cannot create class "org.wavescale.sourcesync.action.ActionProjectConnectionConfig" [Plugin: org.wavescale.sourcesync]
```

Is there a easier, gui interfaced configuration for telling PHPStorm what Java version to use?

This one here isn't as friendly.
https://intellij-support.jetbrains.com/entries/23395793-Configuring-JVM-options-and-platform-properties

**Comment by fioan89** [comment]
Hmmm... it might be one of two issues:
1. You dont have jdk 1.7 or 1.8 as default. I ' ve compiled the plugin targeting 1.7. As @asebak sugested there were some requests for supporting 1.6 but the thing is this version is out of support.
2. There are some prerequisites in the plugin manifest that filters out some jetbrains products. I'll have to check the manifest to see if php storm is supported by the manifest. Probably it is if you installed the plugin through jetbrains repository.

But for now I support the first statement.

**Comment by fioan89** [comment]
Ok so I've ruled out number two from may list. PHPStorm is not filtered out. Back on your question I think eve though you installed java 8, the previous version is still the default one. So if you don't want to mess with jetbrains config files you can try this if are a windows guy https://confluence.atlassian.com/display/DOC/Setting+the+JAVA_HOME+Variable+in+Windows. If you are using linux search on google per your particular distribution.
